pkg/scheduler/util/utils_test.go

```diff
@@ -62,26 +63,70 @@ func newPriorityPodWithStartTime(name string, priority int32, startTime time.Tim
 
 func TestGetEarliestPodStartTime(t *testing.T) {
 	currentTime := time.Now()
-	pod1 := newPriorityPodWithStartTime("pod1", 1, currentTime.Add(time.Second))
-	pod2 := newPriorityPodWithStartTime("pod2", 2, currentTime.Add(time.Second))
-	pod3 := newPriorityPodWithStartTime("pod3", 2, currentTime)
-	victims := &extenderv1.Victims{
-		Pods: []*v1.Pod{pod1, pod2, pod3},
-	}
-	startTime := GetEarliestPodStartTime(victims)
-	if !startTime.Equal(pod3.Status.StartTime) {
-		t.Errorf("Got wrong earliest pod start time")
-	}
-
-	pod1 = newPriorityPodWithStartTime("pod1", 2, currentTime)
-	pod2 = newPriorityPodWithStartTime("pod2", 2, currentTime.Add(time.Second))
-	pod3 = newPriorityPodWithStartTime("pod3", 2, currentTime.Add(2*time.Second))
-	victims = &extenderv1.Victims{
-		Pods: []*v1.Pod{pod1, pod2, pod3},
+	tests := []struct {
+		name              string
+		victims           *extenderv1.Victims
+		expectedStartTime *metav1.Time
+		isDurationTime    bool
```

Comment:
It's hard to test the semantic current time, maybe we can play a trick here. Like we have three pods: 
```
pod1 = newPriorityPodWithStartTime("pod1", 1, currentTime.Sub(time.Second))
pod2 = newPriorityPodWithStartTime("pod1", 1, nil) --> This may need a function refactor here.
pod3 = newPriorityPodWithStartTime("pod1", 1, currentTime.Add(time.Second))
```

Since they have the same priority, pod2.Status.StartTime equals `time.Now`, then pod1 will have the earliest pod start time. Then we can avoid to test the semantic current time.

Reply:
Thanks for the suggestion, I agree with you and make the changes accordingly.